crates/precompiles/src/stablecoin_exchange/dispatch.rs

```diff
@@ -541,7 +540,7 @@ mod tests {
 
     #[test]
     fn stablecoin_exchange_test_selector_coverage() -> eyre::Result<()> {
-        let mut storage = HashMapStorageProvider::new(1).with_spec(TempoHardfork::AllegroModerato);
```

Comment:
do we need to keep the `with_spec` method? i assume it will be used for future hardforks, right?